src/third-version.ts

```diff
@@ -404,14 +413,32 @@ function convertParameter(parameter: any): any {
   });
 
   //Return the new v3 parameter object
-  return Object.assign({...v2ParameterObjectExtensions},
-    enumValues === null ? null : {enum: enumValues},
-    defaultValues === null ? null : {default: defaultValues},
-    description === null ? null : {description},
-    examples === null ? null : {examples},
-    location === null ? null : {location}
-  );
+  return {...v2ParameterObjectExtensions,
+    ...(enumValues === null ? null : {enum: enumValues}),
+    ...(constValue === null ? null : {enum: [constValue]}),
+    ...(defaultValues === null ? null : {default: defaultValues}),
+    ...(description === null ? null : {description}),
+    ...(examples === null ? null : {examples}),
+    ...(location === null ? null : {location})
+  };
 }
+
+/**
+ * This function makes sure we complain if a parameter schema uses now unsupported properties
+ */
+function reportUnsupportedParameterValues(schema: any) {
+  if(schema === undefined) return;
+  const excessProperties = Object.entries(schema).filter((([propertyName,]) => {
+    return !['$ref', 'enum', 'const', 'default', 'examples', 'description'].includes(propertyName);
+  }));
+  if(excessProperties.length > 0) {
+    const listOfProperties = excessProperties.map(([propertyName, property]) => {
+      return `- schema.${propertyName} with value: ${JSON.stringify(property)} are no longer supported`
+    })
+    console.warn(`Found properties in parameter schema that are no longer supported and will be ignored by the converter.\n${listOfProperties.join('\n')}`);
```

Comment:
the thing is that they are not just ignored, person that converts ends up with empty `parameter`,

Reply:
Thats what the sentence mean? 🤔 At least to me 😄

Reply:
```suggestion
    console.warn(`Found properties in parameter schema that are no longer supported. Conversion completes with empty parameter object.\n${listOfProperties.join('\n')}`);
```

wdyt?

Reply:
Fine by me 👍